scripts/rollup/build-all-release-channels.js

```diff
@@ -346,6 +356,14 @@ function updatePackageVersions(
   }
 }
 
+function hashJSFilesInDirectory(directory) {
+  const hash = crypto.createHash('sha1');
+  for (const filePath of glob.sync(directory + '/**/*.js').sort()) {
+    hash.update(fs.readFileSync(filePath));
+  }
+  return hash.digest('hex').slice(0, 8);
+}
+
```

Comment:
Should we hash based on only JS files? Not important now, but hashing on non-directories seems more reliable, in case we will distribute non-js files at some point

Reply:
I'm actually not sure if we even need this version string anyway? I think devtools use them in some capacity, but I'm not sure what the exact requirements are.